files/__addonLocation__/package.json

```diff
@@ -36,31 +39,16 @@
     "@babel/plugin-syntax-decorators": "^7.17.0",
     "@babel/runtime": "^7.17.0",
     "@embroider/addon-dev": "^3.0.0",<% if (typescript) { %>
+    "@glimmer/component": "^1.1.2",
     "@glint/core": "^1.0.2",
     "@glint/environment-ember-loose": "^1.0.2",
     "@glint/template": "^1.0.2",
     "@tsconfig/ember": "^1.0.0",
-    "@types/ember": "^4.0.0",
-    "@types/ember__object": "^4.0.0",
-    "@types/ember__service": "^4.0.0",
-    "@types/ember__controller": "^4.0.0",
-    "@types/ember__string": "^3.16.0",
-    "@types/ember__template": "^4.0.0",
-    "@types/ember__polyfills": "^4.0.0",
-    "@types/ember__utils": "^4.0.0",
-    "@types/ember__runloop": "^4.0.0",
-    "@types/ember__debug": "^4.0.0",
-    "@types/ember__engine": "^4.0.0",
-    "@types/ember__application": "^4.0.0",
-    "@types/ember__test": "^4.0.0",
-    "@types/ember__array": "^4.0.0",
-    "@types/ember__error": "^4.0.0",
-    "@types/ember__component": "^4.0.0",
-    "@types/ember__routing": "^4.0.0",
```

Comment:
typescript scenario is failing. The test-app is still using DT packages, which seem to cause conflicts? Have you seen that?

Reply:
the typescript blueprint we get with `--typescript` is currently wrong, which is causing the  failures. I opened a bunch of PRs over there: https://github.com/typed-ember/ember-cli-typescript/pulls

But if the `--typescript` support switches to using the built in types, that simplifies a lot of things for us as well.

Reply:
More details here: https://github.com/embroider-build/addon-blueprint/pull/123